doc/providers/psr11.rst

```diff
@@ -0,0 +1,105 @@
+PSR-11
+======
+
+The *Psr11ServiceProvider* provides a container for accessing your application
+services using the PSR-11 ``ContainerInterface`` interface.
+
+PSR-11 is a standard that describes a common interface for dependency injection
+containers. Using the PSR-11 container will allow you to:
+
+* Use objects that expect a ``ContainerInterface`` instance.
+
+* Decouple your own code from the ``Silex\Application`` class (something that
+  could prove useful if you intend to port your application to the Symfony
+  full-stack framework in the future).
+
+Services
+--------
+
+* **container**: A container that implements `ContainerInterface
+  <https://github.com/container-interop/fig-standards/blob/master/proposed/container.md>`_
```

Comment:
URLs should be referenced as footnotes (I think that's what we are doing elsewhere)